test/parallel/test-v8-stats.js

```diff
@@ -25,7 +25,8 @@ const expectedHeapSpaces = [
   'old_space',
   'code_space',
   'map_space',
-  'large_object_space'
+  'large_object_space',
+  'read_only_space'
```

Comment:
suggestion: also assert the type below?

Reply:
That already happens, right?

Reply:
I mean add `assert.strictEqual(typeof heapSpace.read_only_space, 'number');` after line 39

Reply:
Oh, that's not what the test is doing. The output looks like this:

```
> v8.getHeapSpaceStatistics()
[ { space_name: 'read_only_space',
    space_size: 0,
    space_used_size: 0,
    space_available_size: 0,
    physical_space_size: 0 },
  { space_name: 'new_space',
    space_size: 4194304,
    space_used_size: 1962224,
    space_available_size: 100112,
    physical_space_size: 3049432 },
  { space_name: 'old_space',
    space_size: 3428352,
    space_used_size: 3343840,
    space_available_size: 552,
    physical_space_size: 3404768 },
  { space_name: 'code_space',
    space_size: 1048576,
    space_used_size: 612928,
    space_available_size: 0,
    physical_space_size: 645696 },
  { space_name: 'map_space',
    space_size: 1073152,
    space_used_size: 641344,
    space_available_size: 80,
    physical_space_size: 667456 },
  { space_name: 'large_object_space',
    space_size: 0,
    space_used_size: 0,
    space_available_size: 1516973568,
    physical_space_size: 0 } ]
```

So, a new object has been added with `space_name` equal to `'read_only_space'`.

Reply:
🤦‍♂️ sorry. Thanks for the explanation